### PR TITLE
[Deleted]

### PR DESCRIPTION
**Pull request by kphoenix137**

### CODE REVIEW

File: Source/items.cpp [hunk 1]
```diff
@@ -714,10 +715,66 @@ int SaveItemPower(const Player &player, Item &item, ItemPower &power)
 		r = RndPL(power.param1, power.param2);
 		item._iPLDam += r;
 		item._iPLToHit += CalculateToHitBonus(power.param1);
+#ifdef _DEBUG
+		switch (power.param1) {
+		case 20:
+			if (item._iPLToHit != 5)
+				!DebugPerfectStat;
```
Comment (StephenCWills) [inline Source/items.cpp:722]:
```suggestion
				DebugPerfectStat = false;
```

File: Source/items.cpp [hunk 2]
```diff
@@ -714,10 +715,66 @@ int SaveItemPower(const Player &player, Item &item, ItemPower &power)
...
+		}
+#endif
 		break;
 	case IPL_TOHIT_DAMP_CURSE:
 		item._iPLDam -= r;
 		item._iPLToHit += CalculateToHitBonus(-power.param1);
+#ifdef _DEBUG
+		switch (power.param1) {
+		case -50:
+			if (item._iPLToHit != 10)
```
Comment (StephenCWills) [inline Source/items.cpp:767]:
```suggestion
			if (item._iPLToHit != -10)
```

File: Source/items.cpp [hunk 3]
```diff
@@ -714,10 +715,66 @@ int SaveItemPower(const Player &player, Item &item, ItemPower &power)
...
+				DebugPerfectStat = false;
+			break;
+		case -25:
+			if (item._iPLToHit != 5)
```
Comment (StephenCWills) [inline Source/items.cpp:771]:
```suggestion
			if (item._iPLToHit != -5)
```

**Comment by AJenbo** [comment]
I'm not sure I see the point of this?

File: Source/items.h [hunk 4]
```diff
@@ -555,6 +555,7 @@ bool ApplyOilToItem(Item &item, Player &player);
 #ifdef _DEBUG
 std::string DebugSpawnItem(std::string itemName);
 std::string DebugSpawnUniqueItem(std::string itemName);
+std::string DebugSpawnPerfectItem(std::string itemName);
```
Comment (StephenCWills) [inline Source/items.h:558]:
```suggestion
std::string DebugSpawnPerfectItem(std::string itemName);
std::string DebugSpawnUniqueItem(std::string itemName);
```

**Review by StephenCWills — CHANGES_REQUESTED** [review]

**Comment by StephenCWills** [comment]
> I'm not sure I see the point of this?

I think it would mainly be useful for testing builds. I don't know that I would argue for merging it, but the PR demonstrates how it can be done.

**Comment by kphoenix137** [comment]
> I'm not sure I see the point of this?

Modder friendly. If someone implements new items, affixes, or revises current items, they will want to test the maximum capabilities of the items for the sake of a creating game balance.

**Pull request closed without merging**